docs/schema-generator/writing-schemas/scalars.md

```diff
@@ -12,7 +13,7 @@ extended scalar types provided by `graphql-java`.
 | `kotlin.Float`      | `Float`      |
 | `kotlin.String`     | `String`     |
 | `kotlin.Boolean`    | `Boolean`    |
-| | **Extended GraphQL Types**       |
+| **Extended GraphQL Types** | **GraphQL Type** |
```

Comment:
I believe graphql-java no longer supports extended types (by default) -> lets double check on it

Reply:
It says these are the current supported scalars

https://www.graphql-java.com/documentation/v15/scalars/

Reply:
Maybe they haven't updated the docs -> https://github.com/graphql-java/graphql-java/releases/tag/v15.0

```
Any special handling for non standard Scalars got removed. This means you have to declare any Long, Char, etc Scalar explicitly in your SDL and provide an implementation.

The implementation for these non standard Scalars are still available in GraphQL Java itself (in the Scalars class) but they will be removed in the next release. The extended scalars project is the recommend way to migrate.
```

PR: https://github.com/graphql-java/graphql-java/pull/1780